### ISSUE TITLE
[Bug] Clicking on a movie/show inside cast&more displays the home screen before going to that selected movie/show

### ISSUE THREAD

**Issue by movianlost** [SerpentDrago/skin.auramod#267]
So I'm in the home screen in Shows, I click on a show, the `information page` opens where you have `play, cast & more, trailer etc`. I click on `cast & more` and go to `Similar` and choose a show and click on it. What happens is that it first displays the home screen for about 1 second, then `please wait` screen appears then the show I selected.

The expected behavior is when you click on a show inside the `cast & more`, the `please wait` screen should appear then the information about that show. There is no need to display the `home screen` again.

For better explanation I have included a video in the link below:

https://user-images.githubusercontent.com/81398427/178169628-155036d5-8e0d-487b-9824-aeb0277b5f0b.mp4

In the above video, the bug appears two times.

The first is I'm in the home screen, I click on `Stranger Things` show, then I click on `cast & more`, then in `Similar` I chose the show `Friends`, here the bug appears as the first home screen that I started the video with appears again, the correct behavior should be `please wait` screen followed by the information page of the show `Friends`.

The second time is when I'm in the information page of `Friends` and I click on `cast & more` and I chose show called `The king of queens`, what happens after is that the `Friends` show's information page appears again, then `please wait` screen then the `The king of queens` show's information page.

**Comment by jojobrogess** [comment]
Yeah this looks like a logical bug with how the timing works with a list in a window focus as it loads in the new window. 

If I'm right, 
I think when you go into a list like that and then select a new item within that list, since our skin is coded to bring up the information view of the selection and since its kinda deep(widget list, file selection brings up information window then goes to cast and more window then to similar list then file selection which then redoes all that and brings up the information window which then intern loads new lists for that) Kodi/addon looses its place a little. Either it's designed to do this or it has to exit out of the current windowed list to enter into the new one. Not sure. 

But that's what you're seeing
It's exiting the list within a list in a window by backing out(or unfocusing the window so that it can focus on another) to the main screen and then entering a new list within a window. 
A broken workaround would be to add in a new focused window during that switch over that acts as a blank loading screen, so that you don't see the switch over. lol but that doesn't fix the problem

It gets even worse when you try to go even deeper like using the next page option, see #232.
Or trying to use the context menu within those windows.
The "deeper" aspects of the information view window(ei: cast&more, similar,recommended) are a little buggy.

But I'm pretty sure the fix inb4after has been working on will smooth this over a bit, since I think the paging aspect of this is why the list within lists gets funky(er) than it should.
